packages/devextreme/js/__internal/scheduler/appointments/m_cell_position_calculator.ts

```diff
@@ -110,6 +110,15 @@ class BaseStrategy {
       inAllDayRow && !this.isVerticalGrouping,
     );
 
+    const groupEdgeIndices = this.viewDataProvider.getGroupEdgeIndices(validGroupIndex);
+    const vMin = this.getCellPosition(
+      {
+        columnIndex: positionByMap.columnIndex,
+        rowIndex: groupEdgeIndices.firstRowIndex,
+      },
+      inAllDayRow && !this.isVerticalGrouping,
+    ).top;
```

Comment:
Let's refactor this code block a little to make it more readable

```suggestion
    const { top: vMin } = this.getCellPosition(
      {
        columnIndex: positionByMap.columnIndex,
        rowIndex: groupEdgeIndices.firstRowIndex,
      },
      inAllDayRow && !this.isVerticalGrouping,
    );
```